conf/application.conf

```diff
@@ -109,7 +109,7 @@ microservice {
       declarationAmendments = enabled
       default = disabled
       merchandiseInBag = disabled
-      newErrorReport = disabled
+      newErrorReport = enabled
```

Comment:
Will this now be enabled in all environments?

Reply:
Yes. Dhanist will be writing the full AC tests for the new Errors screen for this ticket